lib/cypress/app_config.rb

```diff
@@ -40,5 +40,12 @@ def self.[]=(key, val)
       # Allow setting app config during app init, before Rails.cache exists
       clear
     end
+
+    def self.load_yml
+      app_config = YAML.load(ERB.new(File.read("#{Rails.root}/config/cypress.yml")).result)
+      names_config = YAML.load(ERB.new(File.read("#{Rails.root}/config/names.yml")).result)
+      app_config['randomization']['names'] = names_config['names']
```

Comment:
Sorry I should've updated this after speaking with Louis directly, when we spoke I explained my reasoning for breaking it out is due to the fact that the app_config interface is being completely deleted in my pull request in favor of a new Settings model. Also I am bit weary of a setup like this in general since if a value was added to cypress.yml later down the road that happened to conflict with the name `app_config['randomization']['names']` then it would be very confusing to the developer trying to figure out why the rest of the values they added to cypress.yml were showing up but the one with the name `app_config['randomization']['names']` wasn't.